instance/models/instance.py

```diff
@@ -80,13 +80,24 @@ class InstanceState(ResourceState):
     # - delay execution of an operation until the target server reaches a steady state
     # - raise an exception when trying to schedule an operation that depends on a state change
     #   while the target server is in a steady state
-    # Steady states include: Status.Running, Status.ConfigurationFailed, Status.Error, Status.Terminated
-    is_steady_state = False
-
-    # An instance is healthy if it has Status.Running.
+    # Steady states include:
+    # - Status.New
+    # - Status.Running
+    # - Status.ConfigurationFailed
+    # - Status.Error
+    # - Status.Terminated
+    is_steady_state = True
+
+    # An instance is healthy if it is part of a normal (expected) workflow.
     # This information can be used to delay execution of an operation
-    # until the target instance has reached this status.
-    is_healthy_state = False
+    # until the target server has reached a healthy state.
```

Comment:
Nit: I would say that is_healthy_state is more useful as a way "to detect problems and highlight them in the UI or notify users"